source/_components/device_tracker.tomato.markdown

```diff
@@ -29,14 +29,47 @@ device_tracker:
     http_id: YOUR_HTTP_ID
 ```
 
-Configuration variables:
-
-- **host** (*Required*): The IP address of your router, e.g. 192.168.1.1.
-- **username** (*Required*: The username of an user with administrative privileges, usually *admin*.
-- **password** (*Required*): The password for your given admin account.
-- **http_id** (*Required*): The value can be obtained by logging in to the Tomato admin interface and search for `http_id` in the page source code.
+{% configuration %}
+host:
+  description: "The IP address or hostname of your router, e.g. `192.168.1.1` or `rt-ac68u`."
+  required: false
+  type: string
+port:
+  description: "The port number of your router, e.g. `443`."
+  required: false
+  type: int
+  default: 80/443 (ssl disabled/ssl enabled)
```

Comment:
@frenck How should the default port param be handled in the case of ssl/non ssl being chosen?  Not sure how this should be represented to the users.

Reply:
Maybe reference it as "auto detected"?

Of "80/443 (automatically detected)"